apps/desktop-wallet/src/storage/addresses/addressesSelectors.ts

```diff
@@ -33,7 +33,8 @@ export const selectInitialAddress = createSelector(selectAllAddresses, (addresse
   addresses.find((address) => address.index === 0)
 )
 
-export const { selectAll: selectAllContacts } = contactsAdapter.getSelectors<RootState>((state) => state.contacts)
+export const { selectAll: selectAllContacts, selectById: selectContactByHash } =
```

Comment:
I am not sure this is what you want to accomplish. The `ID` of a contact is different than the address hash because the hash can change, but the `ID` cannot. The contacts slice is using `ID` as the entity id. So `selectById` is also using `ID`, not the address hash. There is `makeSelectContactByAddress` below that allows selecting a contact by its hash.